x/ccv/consumer/types/keys.go

```diff
@@ -23,101 +23,150 @@ const (
-	ConsumerRedistributeName = "cons_redistribute"
+	// HistoricalInfoKey is the byte prefix that will store the historical info for a given height
+	HistoricalInfoBytePrefix
 
-	// ConsumerToSendToProviderName is a "buffer" address for outgoing fees to be transferred to the provider chain.
-	ConsumerToSendToProviderName = "cons_to_send_to_provider"
+	// PacketMaturityTimePrefix is the byte prefix that will store maturity time for each received VSC packet
+	PacketMaturityTimeBytePrefix
+
+	// HeightValsetUpdateIDPrefix is the byte prefix that will store the mapping from block height to valset update ID
+	HeightValsetUpdateIDBytePrefix
 
-	// PendingSlashRequestsPrefix is the prefix that will store a list of slash request that must be sent
+	// OutstandingDowntimePrefix is the byte prefix that will store the validators outstanding downtime by consensus address
+	OutstandingDowntimeBytePrefix
+
+	// PendingSlashRequestsPrefix is the byte prefix that will store a list of slash request that must be sent
 	// to the provider chain once the CCV channel is established
-	PendingSlashRequestsPrefix = "pendingslashrequests"
+	PendingSlashRequestsBytePrefix
 
-	// HistoricalInfoKey is the key prefix that will store the historical info for a given height
-	HistoricalInfoKey = "historicalinfokey"
+	// CrossChainValidatorPrefix is the byte prefix that will store cross-chain validators by consensus address
+	CrossChainValidatorBytePrefix
 )
 
-var (
-	// PortKey defines the key to store the port ID in store
-	PortKey                         = []byte{0x01}
-	LastDistributionTransmissionKey = []byte{0x02}
-)
+// Returns the key to the port ID in the store
+func PortKey() []byte {
+	return []byte{PortByteKey}
+}
 
-// UnbondingTimeKey returns the key for storing the unbonding period
-func UnbondingTimeKey() []byte {
-	return []byte(UnbondingTimeKeyString)
+// Returns the key to the last distribution transmission in the store
+func LastDistributionTransmissionKey() []byte {
+	return []byte{LastDistributionTransmissionByteKey}
 }
 
-// ProviderChannelKey returns the key for storing channelID of the provider chain.
-func ProviderChannelKey() []byte {
-	return []byte(ProviderChannelKeyString)
+// UnbondingTimeKey returns the key for storing the unbonding period
+func UnbondingTimeKey() []byte {
+	return []byte{UnbondingTimeByteKey}
 }
 
 // ProviderClientKey returns the key for storing clientID of the provider
 func ProviderClientKey() []byte {
-	return []byte(ProviderClientKeyString)
+	return []byte{ProviderClientByteKey}
+}
+
+// ProviderChannelKey returns the key for storing channelID of the provider chain
+func ProviderChannelKey() []byte {
+	return []byte{ProviderChannelByteKey}
 }
 
 // PendingChangesKey returns the key for storing pending validator set changes
 func PendingChangesKey() []byte {
-	return []byte(PendingChangesKeyString)
+	return []byte{PendingChangesByteKey}
+}
+
+// Returns the historical info key prefix for historical info on each height
```

Comment:
Comments should begin with the element's name they describe.

```suggestion
// HistoricalInfoPrefix returns the historical info key prefix for historical info on each height
```